{"uuid":"7e410847-b9fd-47cd-9f91-bd992e7ea302","historyId":null,"status":"broken","statusDetails":{"message":"Oops...we found an error preparing this test file:\n\n  > cypress\\e2e\\tests\\exam.cy.js\n\n","trace":"\n\nError: Webpack Compilation Error\nModule not found: Error: Package path ./support/constants/allure.js is not exported from package C:\\Users\\julia\\OneDrive\\Documentos\\Desktop\\ZooPlus\\node_modules\\cypress (see exports field in C:\\Users\\julia\\OneDrive\\Documentos\\Desktop\\ZooPlus\\node_modules\\cypress\\package.json)\n    at handle (C:\\Users\\julia\\AppData\\Local\\Cypress\\Cache\\13.4.0\\Cypress\\resources\\app\\node_modules\\@packages\\server\\node_modules\\@cypress\\webpack-preprocessor\\dist\\index.js:212:23)\n    at finalCallback (C:\\Users\\julia\\AppData\\Local\\Cypress\\Cache\\13.4.0\\Cypress\\resources\\app\\node_modules\\webpack\\lib\\Compiler.js:441:32)\n    at C:\\Users\\julia\\AppData\\Local\\Cypress\\Cache\\13.4.0\\Cypress\\resources\\app\\node_modules\\webpack\\lib\\Compiler.js:505:17\n    at C:\\Users\\julia\\AppData\\Local\\Cypress\\Cache\\13.4.0\\Cypress\\resources\\app\\node_modules\\webpack\\lib\\HookWebpackError.js:68:3\n    at Hook.eval [as callAsync] (eval at create (C:\\Users\\julia\\AppData\\Local\\Cypress\\Cache\\13.4.0\\Cypress\\resources\\app\\node_modules\\tapable\\lib\\HookCodeFactory.js:33:10), <anonymous>:6:1)\n    at Hook.CALL_ASYNC_DELEGATE [as _callAsync] (C:\\Users\\julia\\AppData\\Local\\Cypress\\Cache\\13.4.0\\Cypress\\resources\\app\\node_modules\\tapable\\lib\\Hook.js:18:14)\n    at Cache.storeBuildDependencies (C:\\Users\\julia\\AppData\\Local\\Cypress\\Cache\\13.4.0\\Cypress\\resources\\app\\node_modules\\webpack\\lib\\Cache.js:122:37)\n    at C:\\Users\\julia\\AppData\\Local\\Cypress\\Cache\\13.4.0\\Cypress\\resources\\app\\node_modules\\webpack\\lib\\Compiler.js:501:19\n    at Hook.eval [as callAsync] (eval at create (C:\\Users\\julia\\AppData\\Local\\Cypress\\Cache\\13.4.0\\Cypress\\resources\\app\\node_modules\\tapable\\lib\\HookCodeFactory.js:33:10), <anonymous>:6:1)\n    at Hook.CALL_ASYNC_DELEGATE [as _callAsync] (C:\\Users\\julia\\AppData\\Local\\Cypress\\Cache\\13.4.0\\Cypress\\resources\\app\\node_modules\\tapable\\lib\\Hook.js:18:14)\n    at C:\\Users\\julia\\AppData\\Local\\Cypress\\Cache\\13.4.0\\Cypress\\resources\\app\\node_modules\\webpack\\lib\\Compiler.js:498:23\n    at Compiler.emitRecords (C:\\Users\\julia\\AppData\\Local\\Cypress\\Cache\\13.4.0\\Cypress\\resources\\app\\node_modules\\webpack\\lib\\Compiler.js:919:5)\n    at C:\\Users\\julia\\AppData\\Local\\Cypress\\Cache\\13.4.0\\Cypress\\resources\\app\\node_modules\\webpack\\lib\\Compiler.js:490:11\n    at C:\\Users\\julia\\AppData\\Local\\Cypress\\Cache\\13.4.0\\Cypress\\resources\\app\\node_modules\\webpack\\lib\\Compiler.js:885:14\n    at Hook.eval [as callAsync] (eval at create (C:\\Users\\julia\\AppData\\Local\\Cypress\\Cache\\13.4.0\\Cypress\\resources\\app\\node_modules\\tapable\\lib\\HookCodeFactory.js:33:10), <anonymous>:6:1)\n    at Hook.CALL_ASYNC_DELEGATE [as _callAsync] (C:\\Users\\julia\\AppData\\Local\\Cypress\\Cache\\13.4.0\\Cypress\\resources\\app\\node_modules\\tapable\\lib\\Hook.js:18:14)\n    at C:\\Users\\julia\\AppData\\Local\\Cypress\\Cache\\13.4.0\\Cypress\\resources\\app\\node_modules\\webpack\\lib\\Compiler.js:882:27\n    at C:\\Users\\julia\\AppData\\Local\\Cypress\\Cache\\13.4.0\\Cypress\\resources\\app\\node_modules\\neo-async\\async.js:2818:7\n    at done (C:\\Users\\julia\\AppData\\Local\\Cypress\\Cache\\13.4.0\\Cypress\\resources\\app\\node_modules\\neo-async\\async.js:3522:9)\n    at alreadyWritten (C:\\Users\\julia\\AppData\\Local\\Cypress\\Cache\\13.4.0\\Cypress\\resources\\app\\node_modules\\webpack\\lib\\Compiler.js:714:8)\n    at C:\\Users\\julia\\AppData\\Local\\Cypress\\Cache\\13.4.0\\Cypress\\resources\\app\\node_modules\\webpack\\lib\\Compiler.js:802:19\n    at C:\\Users\\julia\\AppData\\Local\\Cypress\\Cache\\13.4.0\\Cypress\\resources\\app\\node_modules\\graceful-fs\\graceful-fs.js:123:16\n    at C:\\Users\\julia\\AppData\\Local\\Cypress\\Cache\\13.4.0\\Cypress\\resources\\app\\node_modules\\@packages\\server\\node_modules\\graceful-fs\\graceful-fs.js:123:16\n    at C:\\Users\\julia\\AppData\\Local\\Cypress\\Cache\\13.4.0\\Cypress\\resources\\app\\node_modules\\@packages\\server\\node_modules\\graceful-fs\\graceful-fs.js:123:16\n    at FSReqCallback.readFileAfterClose [as oncomplete] (node:internal/fs/read_file_context:68:3)\n\nThis occurred while Cypress was compiling and bundling your test code. This is usually caused by:\n\n- A missing file or dependency\n- A syntax error in the file or one of its dependencies\n\nFix the error in your code and re-run your tests."},"stage":"finished","attachments":[],"parameters":[],"labels":[{"name":"suite","value":"exam.cy.js"}],"links":[],"start":null,"name":"Oops...we found an error preparing this test file: exam.cy.js","fullName":"Oops...we found an error preparing this test file: exam.cy.js","stop":null}